### ISSUE TITLE
In settings labels are cut off.

### ISSUE THREAD

**Issue by erperion** [flackbash/AudioAnchor#121]
I would like to point out is that in settings, label and button are in the same line so the label is often cut off because there is not enough space:
Duck audio when pl... [Toggle button]

If you move toggle button under label it won't be cut off:
Duck audio when playing
[Toggle button]

Or you could wrap label to another line:
Duck audio when [Toggle button]
playing

Either way is much more readable.

**Comment by flackbash** [comment]
This is a version problem. For Android versions >= 8.0 Oreo (API 26) the label is wrapped to another line. It should be possible to implement this for previous versions as well (Note to self: see [here](https://stackoverflow.com/questions/9220039/android-preferencescreen-title-in-two-lines)) but it's a bit off a hassle.

**Comment by erperion** [comment]
If it's a hassle then it's not worth fixing it, but maybe creating a wiki page with screenshots or with text overview of each setting would be worth doing?

**Comment by flackbash** [comment]
I agree, it would be good to have some sort of documentation anyhow.
